Show more data in the metrics view

alexkoy opened the pull request:
I have made some changes (especially for our needs) to show more data in the metric-view. It is not configureable to determine how the metric-graphs will be build. It would be comfortable to configure vanity (or a changed behaviour of vanity using flot), so we can see more data in the views. Without this data included, the graphs are at the moment useless for us.

phillbaker commented:
@alexkoy thanks for the PR! It looks like a test is failing when rendering from the command line, could you take a look at it? 

Also, any chance you could include a screenshot of the changes to the dashboard?

alexkoy commented:
I only made some changes especially for our project. These should not be included in the master project. I only wanted to show that the creation of the metric graphs are not configurable. If you have values of about 10-20, the graphs do not show any numbers. I have changed the behaviour in this case to show 0, the highest number and the mean. I prefer a solution, where this can be configured. I also have made another commit to only show integers, because i had some view problems with the graphs and decimals, the number were shown on multiple lines. This should also be configurable.
I also added the sum at the bottom of the graphs, which should be configurable.
It's only an idea to change the master project.

phillbaker commented:
Closing as it looks like this wasn't a PR intended for upstream.
